### ISSUE TITLE
Server List is not showing up

### ISSUE THREAD

**Issue by johenkel** [hexparrot/mineos-node#169]
When logging in to mineos, the list of all servers does not show up. I cannot access the server from within minos. It is up & running though .
(one server up and able to log in  in-game and one server deactiveted atm.)
The webui version was usually shown as well, but it isn't right now.

I was going to add more users that they can set up their own server but with it being not functioning I am unable to.

Do you have any ideas what I can do ?

![screenshot from 2016-02-21 10 46 04](https://cloud.githubusercontent.com/assets/9903140/13203933/7793bb6e-d888-11e5-99e8-e5eb3910e1d0.png)

**Comment by hexparrot** [comment]
First line of defense: [Update your webui](https://minecraft.codeemo.com/mineoswiki/index.php?title=Updating_the_Webui)

**Comment by johenkel** [comment]
Thank you for the quick reply.
I apologize for not mentioning that I am using the mineos jail for FreeNAS.

It does not contain a  "/usr/games/minecraft" folder.
Would that be the "/usr/pbi/mineos-amd64/" folder ?

**Comment by hexparrot** [comment]
This ones tough, because I don't maintain the FreeNAS plugin for MineOS.  It uses my code, sure, but it puts it on a platform that is rife with incompatibilities, namely, that modern NodeJS builds aren't available on FreeNAS 9.3.

See this git issue: https://github.com/hexparrot/mineos-node/issues/95#issuecomment-160770844

TL;DR: nodejs is stuck at 0.10, which is too low to support the functionality MineOS requires in its current state.  In the meantime, FreeNAS only seems to support up to this build: https://github.com/hexparrot/mineos-node/releases/tag/v1.0.1 which is over 100 commits old (and the last 100 commits are mostly bug fixes, likely relevant to what you're experiencing.

**Comment by johenkel** [comment]
I created a new server and that one is showing up in the list .
The old one (that is not in the list) is being started by mineos though.

You might as well close this isssue as it seems to be more a OS/nodejs problem than minos ?!

Thank you for your help !

**Comment by hexparrot** [comment]
My best guess is that the server you can't see is owned by another user--one that you're not using to log into the webui.  Can't tell for certain though, since you also blanked out the username from your screenshot.
